solutions/swb-reference/integration-tests/tests/isolated/projectEnvTypeConfigs/list.test.ts

```diff
@@ -188,4 +190,57 @@ describe('list environment type configs associated to project', () => {
       });
     });
   });
+
```

Comment:
is there a test somewhere checking that ITAdmins can access the data?

Reply:
Will address this in the multistep in a follow up PR.

Reply:
Actually this already exists in multistep/et-etc.test.ts